packages/dmn-js-decision-table/test/spec/features/decision-table-head/editor/DecisionTableHeadEditorSpec.js

```diff
@@ -138,6 +138,24 @@ describe('decision-table-head/editor', function() {
         expect(input.value).to.eql('');
       })
     );
+
+    function hasSpace(name) {
+      return /\s/.test(name);
+    }
+
+    it('should render warning', function() {
+
+      // when
+      const nameEl = domQuery('.output-name', testContainer);
+
+      if (hasSpace(nameEl.textContent)) {
```

Comment:
Why do we need to check this? The test case should assume the element has spaces.

Reply:
I don't quite understand the test work, what if there's no space then the test would fail,  because it doesn't render any validation component and we expect the validation component to exist, something like "cannot expect null to exist"

Reply:
The test should expect the validation to be present. Therefore I'd create the test scenario in this way the text input has spaces before we check the validation.

Reply:
How do I create the test scenario?

Reply:
You could updating the name of the output-name input with spaces and then check for the validation

```js
it('should validate output name', function() {

  // given
  const nameEl = domQuery('.output-name', testContainer);

  // when
  setValue(nameEl, 'with spaces');

  // then
  // do your expectations ...
      
});

```